ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -4184,14 +4196,16 @@ public Set<Partition> getAllPartitionsOf(Table tbl) throws HiveException {
             Hive.get().getConf(), MetastoreConf.ConfVars.BATCH_RETRIEVE_MAX);
     if (batchSize > 0) {
       return getAllPartitionsInBatches(tbl, batchSize, DEFAULT_BATCH_DECAYING_FACTOR, MetastoreConf.getIntVar(
-              Hive.get().getConf(), MetastoreConf.ConfVars.GETPARTITIONS_BATCH_MAX_RETRIES));
+              Hive.get().getConf(), MetastoreConf.ConfVars.GETPARTITIONS_BATCH_MAX_RETRIES), null,
+              false, null, null);
     } else {
       return getAllPartitions(tbl);
     }
   }
 
   public Set<Partition> getAllPartitionsInBatches(Table tbl, int batchSize, int decayingFactor,
-         int maxRetries) throws HiveException {
+         int maxRetries, Map<String, String> partialPartitionSpec, boolean isAuthRequired,
```

Comment:
could create this as a new method and avoid making changes for method calls where this variables are passed as null